README.md

```diff
@@ -1,5 +1,11 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/d122f3edbb6e4ebfa8c74c4b219ce3a2)](https://www.codacy.com/app/Shamanou/FAIRifier?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Shamanou/FAIRifier&amp;utm_campaign=Badge_Grade)
 
+The FAIRifier is a tool to make messy data [FAIR](https://www.dtls.nl/fair-data/).
+
+The FAIRifier is a tool to make data FAIR(Findable, Acessable, Interoperable and Reusable). 
```

Comment:
Repetition of the first line. This should be rewritten to explain more about the tool: where it came from, what it is based on, what changes were made to the original platform, etc.